client/dashboard/profile-wizard/style.scss

```diff
@@ -108,6 +108,41 @@
 		padding-right: 0;
 	}
 
+	.muriel-input-text {
+		&.empty {
+			.components-base-control__label {
+				display: none;
+			}
+
+			input {
+				top: 15px;
+			}
+		}
+
+		input[type='text']:focus {
+			box-shadow: none;
+			border: 0;
+		}
+	}
+
+	.muriel-input-text label,
+	.muriel-select label {
+		display: initial;
+		width: auto;
+		right: auto;
+		left: $gap;
+	}
+
+	.muriel-input-text.active,
+	.muriel-select.active {
+		border: 2px solid $muriel-hot-purple-600;
+
+		input,
+		select {
+			top: 25px;
+		}
+	}
+
```

Comment:
Should we set the default button height here as `48px` (currently `40px`)?

<img width="505" alt="Screen Shot 2019-05-28 at 2 06 00 PM" src="https://user-images.githubusercontent.com/10561050/58454684-c7724680-8151-11e9-931c-79bb0e4320cf.png">